### DISCUSSION TITLE
cannot execute binary file

### DISCUSSION THREAD

**Discussion by chichitaocss** [IIC2133-PUC/2024-2#152] [Ayuda]
Hola, el contenedor está corriendo, pero no puedo ejecutar el programa. Intenté cambiar el Dockerfile sin éxito.
valgrind: ./pegasus: cannot execute binary file
bash: ./pegasus: cannot execute binary file: Exec format error
Ayuda porfa :c

**Comment by sebacarrascop** [comment]
hola, estás usando devcontainers?

**Reply by chichitaocss** [reply]
sí

**Reply by sebacarrascop** [reply]
no me queda claro por qué estás modificando un dockerfile, me puedes decir como estás intentando ejecutar el archivoo?, o escribeme un correo. sebacarrascop@uc.cl
